pkgs/development/python-modules/troposphere/default.nix

```
@@ -0,0 +1,44 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytestCheckHook
+, pythonOlder
+
+  # python dependencies
+, awacs
+, cfn-flip
+, typing-extensions
+}:
+
+buildPythonPackage rec {
+  pname = "troposphere";
+  version = "4.1.0";
+
+  disabled = pythonOlder "3.6";
```

```suggestion
  disabled = pythonOlder "3.7";
```
https://github.com/cloudtools/troposphere/blob/4.1.0/setup.cfg#L43